packages/node-sdk/src/client.ts

```diff
@@ -594,12 +591,13 @@ export class BucketClient {
    * If the company is set, the event will be associated with the company.
    **/
   public async track(
-    userId: string,
+    userId: string | number,
```

Comment:
I wonder if we should declare User Id and CompanyId as types.

And add two check functions that we can reuse across the code (to check their types)